index.js

```diff
@@ -21,7 +22,7 @@ const init = (fn, filePath, data, options) => {
 		sortKeys: false
 	}, options);
 
-	if (options.sortKeys) {
+	if (options.sortKeys && isPlainObj(data)) {
```

Comment:
People are allowed to use any kind of object with `JSON.stringify()`, not just plain objects, so I don't think we should be this strict.

Reply:
Better to use https://github.com/sindresorhus/is-obj

Reply:
`sort-keys` itself uses `is-plain-obj` though so not sure how to go about it?

Reply:
Good point 👍